lib/hot/hot-api.js

```diff
@@ -1,12 +1,19 @@
 import { initProxy } from './proxy';
+import DomAdapter from './proxy-adapter-dom';
+import NativeAdapter from './proxy-adapter-native';
```

Comment:
We can't have native under the same entry point as vanilla svelte because that breaks the loader for non-native apps.

Native proxy code import some modules from TNS that won't be available for non-native users, and webpack will try to resolve everything greedily at compile time. Even if those imports are in a code branch (if) that is never executed at runtime (I've tried require, dynamic import...).

So we do need a dedicated entry point for native... And actually, I think we may even need a dedicated svelte-loader-native package eventually, so that we can correctly declare TNS modules & their versions as peer dependencies in the package.json, without affecting non-native users. On the plus side, that would allow to preconfigure the loader for native context (preprocessor...).